### ISSUE TITLE
[iOS] NativeDeviceOrientation.landscapeRight is DeviceOrientation.landscapeLeft and vice versa

### ISSUE THREAD

**Issue by kaciula** [rmtmckenzie/flutter_native_device_orientation#70]
If I get the current orientation with `await NativeDeviceOrientationCommunicator().orientation()` and then I lock this orientation with `SystemChrome.setPreferredOrientations(<DeviceOrientation>[orientation])` the screen reverses itself. If current orientation was landscapeRight, it becomes landscapeLeft and vice versa. This would mean to me that the plugin wrongly reports the landscape orientations (left vs right). I've tested on iPhone and iPad Mini to confirm this. Not sure if it behaves the same for all iOS devices.

On my Android devices, it behaves as expected.

**Comment by InspectlyMads** [comment]
How did you solve this? Did you just reverse the outputs? :D

**Comment by kaciula** [comment]
@InspectlyMads Yes. For iOS, I reverse landscapeLeft with landscapeRight when setting the orientation.
